iOS Client/DevotionsController/DevotionsCoordinator.swift

```diff
@@ -43,4 +84,14 @@ class DevotionsCoordinator: NSObject, Coordinator, StoreSubscriber {
         build(newRoute: newRoute)
         route = newRoute
     }
+    
+    // MARK: - UNNotificationDelegate
+    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+        let devotions = MBStore().getDevotions()
+        let date = Date()
+        if let devotion = devotions.first(where: { Formatters.devotionDateFormatter.date(from: $0.date) == date}) {
```

Comment:
This is a little dangerous. You're betting that the current date (line 91) exactly matches the devotion's date. Maybe it would be better to convert the current `date` to a string in YYYY-MM-DD format and then do string comparison

Reply:
a moot point, but you could use the new lazy `day` property on devotion instead of doing the Formatters call here . . .

Reply:
`$0.day == date` would be equivalent; but, again, I think we should do string comparison of YYYY-MM-DD instead of date comparison

Reply:
Yeah I think you're right we should compare the date strings not the actually date objects